app/services/set_decline_by_default_to_end_of_cycle.rb

```diff
@@ -0,0 +1,37 @@
+class SetDeclineByDefaultToEndOfCycle
+  def initialize(application_form:)
+    @application_form = application_form
+  end
+
+  def call
+    # Only set Decline by Default (DBD) date if all applications have been acted upon.
+    return if pending_decisions?
```

Comment:
Is this still relevant in the context of continuous applications?

Reply:
assumption is correct! We only need to worry about the single `application_choice` that has received an offer. Because of this, i think we could just pass the single `application_choice` into this class, rather than an `application_form`